recipes/smallgenomeutilities/0.3.1/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
you can move this into the meta.yaml under build/scripts ... and please use pip install as in the other python packages. Thanks you.

Reply:
Done, waiting for the online cloud CircleCI (local Circle on my workstation was happy already)